packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/attributeStartingAt..st

```diff
@@ -0,0 +1,10 @@
+attributes
+attributeStartingAt: aMarkdownDelimiterEmphasis
```

Comment:
We don't understand. Please elaborate

Reply:
The MarkdownDelimiterEmphasis-Object is supposed to be a delimiter rather than an emphasis, right? Like, if I want to imagine the object, its a delimiter with a content (* or ** or whatever) and is handled as a token for this content. Its effect afterwards is the text emphasis, but the object itself is not an emphasis. So I mean, I would consider renaming the whole class into MarkdownEmphasisDelimiter or MarkdownDelimiterForEmphasis, because just having DelimiterEmphasis makes it hard to understand what the object is supposed to be or to do

Reply:
I think once we have more Delimiter Types it will make sense to be honest. Also I really want Delimiter to come first in the name because it we have MarkdownEmphasisDelimiter and MarkdownLinkDelimiter it would be weird to have Delimiter at the end. Also I don't think having a For in the name would make a difference. It kind of clutters the name. Thus I would like it to stay like it is.